### ISSUE TITLE
Unclear Example in Techtonica Slide deck

### ISSUE THREAD

**Issue by aedward8** [Techtonica/curriculum#1533]
### Page where problem found?

https://github.com/Techtonica/curriculum/blob/main/react-js/react-part-2-component-state.md

slide deck link - https://docs.google.com/presentation/d/1Trj_7vWYe79_mDboDxN9rdT94yjY5Dt55xtU1k3dUsw/edit?usp=sharing

### Type of problem

Common examples:

- Confusing or incomplete explanation
The example on slide 16 of this lesson was extremely hard to follow. I was especially confused on where the "props" was being sent to or from since I didn't see the 'props' keyword used at all. I think a simple explanation would work.

### Suggested Solution
An explanation such as

We are send the child Message component the prop ( message = 'hello" ) from the App parent react component. The part in parenthesis is the actual prop we are sending.

**Comment by vegetabill** [comment]
Since we're changing the React curriculum to focus mostly on functional components, I'm guessing these examples will be mostly rewritten so I'm going to close this in the meantime but thanks for reporting @aedward8
